### DISCUSSION TITLE
TLS Mutual authentication (2 way SSL authentication )

### DISCUSSION THREAD

**Discussion by fittachiz** [uNetworking/uWebSockets#1460] [Q&A]
Hi
I looked at "SocketContextOptions," but I couldn't find a way to verify clients.
I'm looking for a Nodejs equivalent of "rejectUnauthorized." [(tls.createServer)](https://nodejs.org/api/tls.html#tlscreateserveroptions-secureconnectionlistener)
Thank you very much.

**Comment by MikkoAtWork** [comment]
I'm interested in mutual TLS too. I noticed that if SocketContextOptions has defined ca_file_name, then the SSL_VERIFY_PEER is enabled in SSL (https://github.com/uNetworking/uSockets/blob/af57ff731eea55094c3de53d26716e107c12933b/src/crypto/openssl.c#L470)
Could you confirm that @alexhultman ?
I haven't tested that yet but probably going to give it a try within couple of weeks.
